sly.el

```diff
@@ -1395,6 +1395,14 @@ Fall back to `sly-init-using-slynk-loader' if ASDF fails."
 (defun sly-read-port-and-connect (inferior-process)
   (sly-attempt-connection inferior-process nil 1))
 
+(defcustom sly-connection-update-interval 0.3
+  "Interval at which the connection is polled when first connecting.
```

Comment:
`Seconds to wait between connection attempts when first connecting.`

No more docstring verbiage is needed.